src/popups/HandicapPopup/HandicapPopupElement.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import { compose } from 'lodash/fp';
+
+const propTypes = {
+  t: PropTypes.func.isRequired,
+  properties: PropTypes.shape.isRequired,
+  k: PropTypes.string.isRequired,
```

Comment:
What is the meaning of `k` and `t` ?

Reply:
I got inspired by someone else... ;-)
I agree with you. The handicap popup is more a draft and it will definitely not stay like this. However, `k` should be renamed to `key`.